src/App.jsx

```diff
@@ -14,70 +15,66 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [food, setFood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {food.length !== 0 ? (
+        /* eslint-disable */
+        <h1 className="title is-flex is-align-items-center">
+          {`${food} is selected`}
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => setFood('')}
+          />
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+      )}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+      {goods.map(good => (
+        <table className="table">
+          <tbody>
+            {food === good ? (
+              <tr data-cy="Good" className="has-background-success-light">
+                <td>
+                  <button
+                    data-cy="RemoveButton"
+                    type="button"
+                    className="button is-info"
+                    onClick={() => setFood('')}
+                  >
+                    -
+                  </button>
+                </td>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+                <td data-cy="GoodTitle" className="is-vcentered">
+                  {good}
+                </td>
+              </tr>) : (
```

Comment:
Again, in both case you render `<tr data-cy="Good">`. Just put ternary operator inside and render elements/add classes conditionally